dcm4che-imageio/src/main/java/org/dcm4che3/imageio/stream/ImageInputStreamAdapter.java

```diff
@@ -38,62 +38,28 @@
 
 package org.dcm4che3.imageio.stream;
 
+import org.dcm4che3.io.stream.CloseableImageInputStreamAdapter;
+
 import java.io.IOException;
 import java.io.InputStream;
 
 import javax.imageio.stream.ImageInputStream;
 
 /**
+ * ImageInputStream which does not close the underlying stream.
  * @author Gunter Zeilinger <gunterze@gmail.com>
- *
+ * @deprecated ImageInputStreams should always be closed.  If an independant delegate is required then wrap a
+ * DelegateImageInputStream with CLoseableImageInputStreamAdapter.
```

Comment:
Typo in class name: CLoseableImageInputStreamAdapter
Would be good to do actual links to the classes with {@link ...}.